### ISSUE TITLE
Cucumber incompatible issue

### ISSUE THREAD

**Issue by MrBokeh** [rmpestano/cukedoctor#87]
The line below is for very old version of Cucumber for Java:
@CucumberOptions(plugin = {"json:target/cucumber.json"} )

Since the newer version of Cucumber does not support the plugin tag but instead using the format tag which is written like this:
@RunWith(Cucumber.class)
@Cucumber.Options(format = {"pretty", "html:target/cucumber-htmlreport","json-pretty:target/cucumber.json"})

But it will cause no result warning like below and the generated documentation has no test result and all causes thumbs down:
Step is missing Result: Given
Step is missing Result: When
Step is missing Result: Then
.... etc

If I run the maven build with my project and mvn cukedocker separately the result seems not recorded into the JSON report, please advise.

**Comment by rmpestano** [comment]
Hi @MrBokeh, what version of cucumber are you using? As I see in official docs the **plugin** attribute is still the way to confugure features. See here in "Configuration" section: https://cucumber.io/docs/reference/jvm#java

**Comment by MrBokeh** [comment]
From memory, It's v1.0.14. Is there a specific way to configure Cucumber or cukedoctor in the maven pom.xml file to enable the result log to the json file?

On 2 Mar 2017, at 8:38 PM, Rafael M. Pestano <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @MrBokeh<https://github.com/MrBokeh>, what version of cucumber are you using? As I see in official docs the plugin attribute is still the way to confugure features. See here in "Configuration" section: https://cucumber.io/docs/reference/jvm#java

**Comment by rmpestano** [comment]
> From memory, It's v1.0.14. Is there a specific way to configure Cucumber or cukedoctor in the maven pom.xml file to enable the result log to the json file?

No, it's an application configuration, Cukedoctor just expects the json is already built, it will scan build directory and generate documentation based on cucumber json files it finds .

for v1.0.14 you will have to use the **format** attribute as in your first comment. 

> If I run the maven build with my project and mvn cukedocker separately the result seems not recorded into the JSON report, please advise. 
Do you have a simple (maven) project which reproduces your problem?

**Comment by MrBokeh** [comment]
What's your email address?

Sent from my iPhone

On 2 Mar 2017, at 8:59 PM, Rafael M. Pestano <notifications@github.com<mailto:notifications@github.com>> wrote:


From memory, It's v1.0.14. Is there a specific way to configure Cucumber or cukedoctor in the maven pom.xml file to enable the result log to the json file?

No, it's an application configuration, Cukedoctor just expects the json is already built, it will scan build directory and generate documentation based on cucumber json files it finds .

for v1.0.14 you will have to use the format attribute as in your first comment.

If I run the maven build with my project and mvn cukedocker separately the result seems not recorded into the JSON report, please advise.
Do you have a simple (maven) project which reproduces your problem?

**Comment by rmpestano** [comment]
You can find it on my profile (rmpestanoATgmail)

**Comment by MrBokeh** [comment]
Sent!


On 2 March 2017 at 9:06:17 PM, Rafael M. Pestano (notifications@github.com<mailto:notifications@github.com>) wrote:

You can find it on my profile (rmpestanoATgmail)

**Comment by rmpestano** [comment]
Hi, the json file is being generated in target folder so your configuration is ok. 

I was able to generate the documentation but as all tests are failing here I also got the thumbs down.

Note that as the tests was failing the maven execution stoped ([see here](http://stackoverflow.com/questions/30759612/force-execution-of-a-maven-phase)) and documentation was not generated so I had to run mvn cukedoctor:execute separately.
